FakeTrello/Models/Board.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Board
+    {
+        [Key]
+        public int BoardId { get; set; } //Primary key
+
+        public string Name { get; set; } //Board Name
+
+        public string URL { get; set; } //Board URL
+
+        public TrelloUser Owner { get; set; } // 1 to 1 relationship
```

Comment:
Do your Cards allow multiple contributors? What would you add to this class to have a contributors?